include/curl-session.h

```diff
@@ -78,7 +78,8 @@ class CurlSession : public CurlCallbacks
     }
 
 
-    void addFormFile(const std::string& key, const std::string& filePath, const std::string& mimeType);
+    void addFormFile(const char* key, const char* filePath, const char* mimeType);
```

Comment:
The idea is to avoid creating temp std::string object just to take its buffer with c_str() inside the method. I am not ready to say, if compiler can do such optimization and avoid creating string object.